python/docs/librmm_docs/deprecated.rst

```diff
@@ -0,0 +1,5 @@
+Deprecated functionality
+========================
+
+.. doxygenpage:: deprecated
+   :content-only:
```

Comment:
The addition of `@deprecated` tags produces a new page of doxygen output that we must include somewhere in the rst toctree such that sphinx will find the link anchors.